lib/node_modules/@stdlib/math/base/special/rsqrt/manifest.json

```diff
@@ -1,78 +1,78 @@
 {
-	"options": {
-		"task": "build"
-	},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"task": "build",
-			"src": [
-				"./src/rsqrt.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": [
-				"@stdlib/math/base/napi/unary",
-				"@stdlib/math/base/special/sqrt"
-			]
-		},
-		{
-			"task": "benchmark",
-			"src": [
-				"./src/rsqrt.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": [
-				"@stdlib/math/base/special/sqrt"
-			]
-		},
-		{
-			"task": "examples",
-			"src": [
-				"./src/rsqrt.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": [
-				"@stdlib/math/base/special/sqrt"
-			]
-		}
-	]
+    "options": {
```

Comment:
Per our EditorConfig configuration, please use two-space indentation in all JSON files. See https://github.com/stdlib-js/stdlib/blob/183d056687e023d43cded450f0d9e4739aaff8bd/.editorconfig#L46.